src/style.css

```
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir-Book" ;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+body {
+  margin: 0 8px;
+}
+
+.searchBar {
+  display: block;
+  margin: 20px 0;
+}
+
+.Big-searchBar {
```

avoid using big letters for namings - be consistent
```suggestion
.big-searchBar {
```